git.go

```diff
@@ -20,6 +20,8 @@ import (
 	"sigs.k8s.io/kustomize/api/types"
 )
 
+// GitOperator is our wrapper aroud go-git to do GitOps, and
+// tagging commits to correlate them with the container image tags.
```

Comment:
I think the correct move is to tag the commit, or rather, to create a pull request by rewriting the image tag written in kustomization.yaml with the latest commit hash. [like this](https://github.com/zaiminc/gocat/blob/d697bf0bb8cdc44b9357a51be9317aaf19cbdabc/git.go#L262)

Reply:
```suggestion
// adding/tagging/pushing a commit to update the image tag(s) in the kustomization.yaml.
```

@pirlodog1125 Thank you very much for your comment! Does this sound better?

Reply:
Yes, that sounds much better! Thank you for